sdk/terraform/Azure.ResourceManager.Terraform/CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.1.0-beta.1 (2025-06-19)
```

Comment:
we have never released 1.0.0 therefore the next available version for release should be `1.0.0-beta.2`. Please change the version number back.

Reply:
Fixed